autogenerated.ts

```diff
@@ -0,0 +1,14 @@
+// This file is meant to be replaced by the MakeCode extension,
+// but in the meantime, it contains a usable example model and labels
+enum MlRunnerLabels {
+    //% block="Shake"
+    Shake = 1,
+    //% block="Still"
+    Still = 2,
+    //% block="Circle"
+    Circle = 3,
+};
+
+function getModelBlob(): Buffer {
```

Comment:
Moved the `getModelBlob()` definition from pxtextension.ts to here, but I'm guessing we need to delete this file completely so that when the MakeCode editor extension creates a new .ts file with this data the function doesn't get redifined?

Reply:
I think it needs to be declared in a file we don't dyanmically create, so we have `let getModelBlob;` in pxtextension.ts and then we define it in the code we generate. Yes, this file we can't keep.

Reply:
As a test building locally, if I leave the `autogenerated.ts` file and the `let getModelBlob: () => Buffer;` declaration in `pxtextension.ts` I got the following error:
```
error TS2451: Cannot redeclare block-scoped variable 'getModelBlob'
```
Maybe the syntax I am using in `autogenerated.ts` is different to the syntax used by the editor extension file.
Would you be able to push an example of the autogenerated file that the editor extension can create and add it to this repo? If we don't add the file to pxt.json I shouldn't be included by default and it would help me test locally.

Reply:
The syntax is slightly different for the function assignment. `autogenerated.ts` now contains this code.